blocks/leadspace/leadspace.css

```diff
@@ -138,36 +157,34 @@ main div.leadspace-wrapper {
     width: 320px;
 }
 
-/* Image Variants */
-.leadspace > div:first-child > div:nth-child(2) picture {
-    display: inline-block;
-    align-self: flex-end;
-}
-
 /* stylelint-disable-next-line no-descending-specificity */
-.leadspace.event picture {
+
+.leadspace.event > div:first-child > div:nth-child(2) picture {
```

Comment:
Are there other pictures you are ensuring aren't set to none with this selector?

Reply:
I don't see any Figma designs with more than 1 picture in that div on events pages.